templates/EFS.yml

```diff
@@ -113,8 +113,12 @@ Resources:
               - '*'
```

Comment:
Seems to be relevant for website enabled buckets only?

Reply:
Deleted